src/main/java/com/basilisk/backend/presenters/LoginPresenter.java

```diff
@@ -37,4 +37,24 @@ public boolean loginUser(String username, String password) {
 
 
     }
+
+    //Added sign up presenter
+    public boolean signupUser(String name, String username, String password) {
+        LOGGER.info("Signup Attempt: Username = " + username);
+
+        User user = userService.retrieveUserByUserName(username);
```

Comment:
This needs to retrieveUserByUsername but ignoring the case of username so Markus and markus should be the same name

Reply:
Also make sure to not allow username, password and name to be empty ("")